tests/mdev_configuration_allocation_test.go

```diff
@@ -354,6 +360,174 @@ var _ = Describe("[Serial][sig-compute]MediatedDevices", Serial, decorators.VGPU
 			Eventually(checkAllMDEVCreated(newDesiredMdevTypeName, newExpectedInstancesNum), 3*time.Minute, 15*time.Second).Should(BeInPhase(k8sv1.PodSucceeded))
 
 		})
+		It("should successfully create VMI when guest memory is less than or equal to request memory", func() {
```

Comment:
The 3 tests you're adding under `MediatedDevices` don't seem to have anything to do with mediated devices, or did I miss something?
If they don't, this is the wrong place for them.